lib/zstd.h

```diff
@@ -828,8 +828,23 @@ ZSTDLIB_API size_t ZSTD_freeDStream(ZSTD_DStream* zds);  /* accept NULL pointer
  *     ZSTD_DCtx_reset(zds, ZSTD_reset_session_only);
  *     ZSTD_DCtx_refDDict(zds, NULL);
  */
-ZSTDLIB_API size_t ZSTD_initDStream(ZSTD_DStream* zds);
-
+ZSTDLIB_API size_t ZSTD_initDStream(ZSTD_DStream* zds); /**< Start new decompression operation */
```

Comment:
Rather than adding an additional comment, you should modify the comment block above this function so that it appears in the documentation (I guess by adding a first line like `/*! ZSTD_initDStream() :`).

Reply:
Ok, I'll update the PR. Thanks for the suggestion